Makefile

```diff
@@ -1,2 +1,3 @@
 test:
 	pep8 --exclude=env --max-line-length=120 .
+	python ./manage.py test --settings=zosia16.settings.test
```

Comment:
Uwaga: teoretycznie to jest dobre, ale tak czy inaczej najpierw wykona się plik settings z `manage.py` a dopiero potem ten `zosia16.settings.test`. Czyli może pojawić się gdzieś przy odpalaniu testów błąd w stylu `cannot connect to database` bo ktoś nie będzie miał odpalonego postgresa :)